core/core_integrations/facebook.js

```diff
@@ -127,35 +127,35 @@ exports.start = function(callback) {
             }
 
             switch (event.type) {
-                case 'message': {
-                    getSenderName(api, event, function(name) {
-                        var data = shim.createEvent(event.threadID, event.senderID, name, event.body + '');
-                        callback(platform, data);
-                    });
-                    break;
-                }
-                case 'event': {
-                    switch (event.logMessageType) {
-                        case 'log:unsubscribe': {
-                            var usrs = event.logMessageData.removed_participants;
-                            for (var i = 0; i < usrs.length; i++) {
-                                usrs[i] = usrs[i].split(':')[1];
-                                if (threadInfo[event.threadID] && threadInfo[event.threadID][usrs[i]]) {
-                                    delete threadInfo[event.threadID][usrs[i]];
-                                }
-                            }
-                            break;
-                        }
-                        case 'log:subscribe': {
-                            var usrs = event.logMessageData.added_participants;
-                            for (var i = 0; i < usrs.length; i++) {
-                                usrs[i] = usrs[i].split(':')[1];
-                            }
-                            getSenderInfo(usrs, api, event, function(){});
-                            break;
+            case 'message': {
+                getSenderName(api, event, function(name) {
+                    var data = shim.createEvent(event.threadID, event.senderID, name, event.body + '');
+                    callback(platform, data);
+                });
+                break;
+            }
+            case 'event': {
+                switch (event.logMessageType) {
+                case 'log:unsubscribe': {
+                    var usrs = event.logMessageData.removed_participants;
+                    for (var i = 0; i < usrs.length; i++) {
+                        usrs[i] = usrs[i].split(':')[1];
+                        if (threadInfo[event.threadID] && threadInfo[event.threadID][usrs[i]]) {
+                            delete threadInfo[event.threadID][usrs[i]];
                         }
                     }
                     break;
+                    }
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 16 space characters but found 20.](https://www.codacy.com/app/theprojectshop/Kassy/file/2651361698/issues/source?bid=3282987&fileBranchId=3324570#l148)